App Analytics migration instructions (#13472)
DataDog/documentation: content/en/tracing/legacy_app_analytics/_index.md

This page describes deprecated features with configuration information relevant to legacy App Analytics, useful for troubleshooting or modifying some old setups. To have full control over your traces, use <a href="/tracing/trace_ingestion">ingestion controls</a> and <a href="/tracing/trace_retention">retention filters</a> instead.
</div>

### Move away from the legacy setup

Navigate the [ingestion control page][1] to spot services with legacy configuration. These are flagged with a `Legacy Setup` status.

> To move away from App Analytics, remove every [configuration option](#app-analytics-setup) related to this setup. Instead, use our agent and libraries [sampling mechanisms][2] that ensure to keep complete traces.

Reviewer: ```suggestion
To move away from the legacy App Analytics setup, remove every [configuration option](#app-analytics-setup) related to it. Instead, use the Datadog Agent and tracing libraries [sampling mechanisms][2] that ensure to keep complete traces.
```
